sc_rf_service/sc_rf_service.py

```diff
@@ -39,53 +221,95 @@ class LiquidLevelPVGroup(PVGroup):
-    nrp = pvproperty(value=0, name="NRP:STATSUMY", dtype=ChannelType.DOUBLE,
-                     record="ai")
+    nrp = pvproperty(
+        value=0, name="NRP:STATSUMY", dtype=ChannelType.DOUBLE, record="ai"
+    )
     aact_mean_sum = pvproperty(value=0, name="AACTMEANSUM")
     # TODO - find this and see what type pv it is on bcs/ops_lcls2_bcs_main.edl
     bcs = pvproperty(value=0, name="BCSDRVSUM", dtype=ChannelType.DOUBLE)
 
 
 class CryoPVGroup(PVGroup):
-    uhl = pvproperty(value=0, name="LVL.SEVR", dtype=ChannelType.ENUM,
-                     enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"))
+    uhl = pvproperty(
+        value=0,
+        name="LVL.SEVR",
+        dtype=ChannelType.ENUM,
+        enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"),
+    )
 
 
 class HOMPVGroup(PVGroup):
-    upstreamHOM = pvproperty(value=0, name="18:UH:TEMP.SEVR", dtype=ChannelType.ENUM,
-                             enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"))
-    downstreamHOM = pvproperty(value=0, name="20:DH:TEMP.SEVR", dtype=ChannelType.ENUM,
-                               enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"))
+    upstreamHOM = pvproperty(
+        value=0,
+        name="18:UH:TEMP.SEVR",
+        dtype=ChannelType.ENUM,
+        enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"),
+    )
+    downstreamHOM = pvproperty(
+        value=0,
+        name="20:DH:TEMP.SEVR",
+        dtype=ChannelType.ENUM,
+        enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"),
+    )
 
 
 class RACKPVGroup(PVGroup):
-    hwi = pvproperty(value=0, name="HWINITSUM", dtype=ChannelType.ENUM,
-                     enum_strings=("Ok", "HW Init running", "LLRF chassis problem"),
-                     record="mbbi")
-    fro = pvproperty(value=0, name="FREQSUM", dtype=ChannelType.ENUM,
-                     enum_strings=("OK", "Still OK", "Faulted"))
+    hwi = pvproperty(
+        value=0,
+        name="HWINITSUM",
+        dtype=ChannelType.ENUM,
+        enum_strings=("Ok", "HW Init running", "LLRF chassis problem"),
+        record="mbbi",
+    )
+    fro = pvproperty(
+        value=0,
+        name="FREQSUM",
+        dtype=ChannelType.ENUM,
+        enum_strings=("OK", "Still OK", "Faulted"),
+    )
     fscan_start = pvproperty(value=0, name="FSCAN:FREQ_START")
     fscan_stop = pvproperty(value=0, name="FSCAN:FREQ_STOP")
     fscan_thresh = pvproperty(value=0, name="FSCAN:RMS_THRESH")
     fscan_overlap = pvproperty(value=0, name="FSCAN:MODE_OVERLAP")
-    prl = pvproperty(value=0, name="PRLSUM.SEVR", dtype=ChannelType.ENUM,
-                     enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"))
-    pjt: PvpropertyDouble = pvproperty(value=0, name="PRLJITSUM",
-                                       dtype=ChannelType.DOUBLE)
+    prl = pvproperty(
+        value=0,
+        name="PRLSUM.SEVR",
+        dtype=ChannelType.ENUM,
+        enum_strings=("NO_ALARM", "MINOR", "MAJOR", "INVALID"),
+    )
```

Comment:
Is this the same PV as the PRLSUM fault PV in line 514 in the CavFaultPVGroup class?

Reply:
Oh! Never mind. We changed the PRLSUM pv from a rack PV to a cavity PV. This one's the rack PV, and the other one in line 514 looks like the updated cavity one